src/Buzz/OAuthBrowser.php

```diff
@@ -17,11 +19,23 @@ class OAuthBrowser extends Browser
     private $isAuthenticated = false;
 
     /** @var array */
-    private $headers = array();
+    private $headers;
 
     /** @var string */
     private $baseUri;
 
+    public function __construct(ClientInterface $client = null, FactoryInterface $factory = null)
+    {
+        parent::__construct($client, $factory);
+
+        $composerJson = file_get_contents(__DIR__ . "/../../composer.json");
```

Comment:
Reading and decoding the file every time is a no go for me, a hardcoded constant would be better and fail safe.